cpu/cc2538/dev/sys-ctrl.c

```diff
@@ -70,22 +70,22 @@ sys_ctrl_init()
    * 32KHz source: RC or crystal, according to SYS_CTRL_OSC32K_USE_XTAL
    * System Clock: 32 MHz
    * Power Down Unused
-   * I/O Div: 16MHz
-   * Sys Div: 16MHz
+   * I/O Div: according to SYS_CTRL_IO_DIV
+   * Sys Div: according to SYS_CTRL_SYS_DIV
    * Rest: Don't care
    */
 
   val = SYS_CTRL_OSCS | SYS_CTRL_CLOCK_CTRL_OSC_PD
-      | SYS_CTRL_CLOCK_CTRL_IO_DIV_16MHZ | SYS_CTRL_CLOCK_CTRL_SYS_DIV_16MHZ;
+    | SYS_CTRL_IO_DIV | SYS_CTRL_SYS_DIV;
   REG(SYS_CTRL_CLOCK_CTRL) = val;
 
   while((REG(SYS_CTRL_CLOCK_STA) & (SYS_CTRL_CLOCK_STA_OSC32K |
-        SYS_CTRL_CLOCK_STA_OSC)) != SYS_CTRL_OSCS);
+                                    SYS_CTRL_CLOCK_STA_OSC)) != SYS_CTRL_OSCS) ;
```

Comment:
Unwanted cosmetic change: it makes the line longer than 80 characters, and there must be no space before semicolons. See:
https://github.com/contiki-os/contiki/blob/master/doc/code-style.c